typescript/api-cors-lambda-crud-dynamodb/lambdas/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "lambda-crud-dynamodb",
+  "version": "1.0.0",
+  "description": "Lambdas to do CRUD operations on DynamoDB",
+  "private": true,
+  "license": "MIT",
+  "devDependencies": {
+    "@types/node": "*",
+    "@types/uuid": "*"
+  },
+  "dependencies": {
+    "aws-sdk": "*",
+    "uuid": "*"
+  }
+}
```

Comment:
Strange this should have a newline....

Reply:
For some reason I cannot see the newline in GitHub's review UI. However when I click "edit file" I see there is already a new line. LMK if you don't see it during your review @NGL321.

Reply:
Did some digging into this and it looks like every file does have a newline char added however you'll only see it in your local editor or when editing the file via GitHub's "edit file" UI. If you want the newline to render in GitHubs "view file" view, then you'll have to add 2 newlines to the bottom of the file.

Is this alright as is? @NGL321